src/routes.ts

```diff
@@ -295,6 +295,7 @@ export const getRoutes = ({
   if (!disabledFeatures.includes(Features.KILL_ENDPOINT)) {
     router.get('/kill/all', async (_req, res) => {
       await chromeHelper.killAll();
+      puppeteerProvider.setSwarm(await puppeteerProvider.start());
```

Comment:
Sorry for the churn on this one. Why don't we just make `puppeteerProvider.start` "reset" the `chromeSwarm` array anytime it's called? Maybe even rename it so it's more obvious that it's rotating the chrome instances it caches. Just trying to make it as idiomatic as possible.